data/conquest_episode_names.csv

```diff
@@ -0,0 +1,77 @@
+episode_id,local_language_id,name
```

Comment:
I see that there's a lot of data in here, I think these are like the chapter names for the Conquest spin-off games, do we need all of this extra data?

Reply:
I just included all of it because it was kinda hard to determine what was needed and what wasn't necessary.

Reply:
gotcha, maybe we can open a new issue to prune some of this data later then